profile-controller/rockcraft.yaml

```diff
@@ -1,9 +1,9 @@
-# Constructed from https://github.com/kubeflow/kubeflow/blob/v1.7-branch/components/profile-controller/Dockerfile
+# Constructed from https://github.com/kubeflow/kubeflow/blob/v1.8.0/components/profile-controller/Dockerfile
 
 name: profile-controller
 summary: Controller for Kubeflow Profile objects
 description: Controller for Kubeflow Profile objects
-version: v1.7.0_22.04_1 # version format: <KF-upstream-version>_<Charmed-KF-version>
+version: "1.8.0"
 
 # TODO: use base: bare
 base: ubuntu:22.04
```

Comment:
@TakoB222 please sure to use a base in the format of `ubuntu@22.04`

Here is an example of this:
https://github.com/canonical/kubeflow-rocks/blob/main/jupyter-web-app/rockcraft.yaml#L9